root/static/scripts/edit/URLCleanup.js

```diff
@@ -3699,6 +3699,30 @@ export const CLEANUPS: CleanupEntries = {
       return {result: false, target: ERROR_TARGETS.URL};
     },
   },
+  'allocine': {
+    hostname: 'allocine.fr',
+    match: [/^(https?:\/\/)?(?:www\.)?allocine\.fr\//i],
+    restrict: [LINK_TYPES.otherdatabases],
+    clean(url) {
+      return url.replace(/^https?:\/\/(?:www\.)?(.*)$/, 'https://$1');
```

Comment:
It seems like it would make sense to clean up things like `https://www.allocine.fr/personne/fichepersonne-38022/filmographie/` into the same link rather than just rejecting them. Amusingly, `https://www.allocine.fr/personne/fichepersonne-38022` does not actually work at all and it *does* need to be turned into `https://www.allocine.fr/personne/fichepersonne_gen_cpersonne=38022.html`. Such a weird decision.

Same for a link with more stuff such as `https://www.allocine.fr/personne/fichepersonne-38022/filmographie/#technical`